src/alchemlyb/convergence/convergence.py

```diff
@@ -20,10 +23,12 @@ def forward_backward_convergence(df_list, estimator='mbar', num=10):
     ----------
     df_list : list
         List of DataFrame of either dHdl or u_nk.
-    estimator : {'mbar', 'bar', 'ti', 'autombar'}
+    estimator : {'MBAR', 'BAR', 'TI'}
         Name of the estimators.
```

Comment:
add deprecation note
```suggestion
        Name of the estimators.
        See the important note below on the use of "MBAR".

        .. deprecated:: 1.0.0
           Lower case input is also accepted until release 2.0.0.
```